### ISSUE TITLE
Tables in Appendices Aren't Numbered and Title May Not Be Displaying Correctly

### ISSUE THREAD

**Issue by uskka** [Connexions/webview#1797]
Description :  Table description appears as grey, and there is no table numbering but the book is baked. Similar issue appeared in unbaked book:  https://github.com/Connexions/cnx-recipes/issues/392

https://katalyst01.cnx.org/contents/Ax2o07Ul@11.32:cjQyhWg-@7/Useful-Information

![screenshot-katalyst01 cnx org-2018 08 03-09-43-15](https://user-images.githubusercontent.com/40228918/43630822-b4832cdc-9701-11e8-989f-11275ba2623b.png)

**Comment by PiotrKozlowski** [comment]
Tables from this link does not have `.os-table` class. 

Also first of those tables should be swipable but it can't without `.os-table` class

**Comment by helenemccarron** [comment]
Mhh. If the book is baked properly, it might be a recipe issue. Are tables working properly in the full baked html?

**Comment by wilkus27** [comment]
The same problem in College Physics for AP : https://katalyst01.cnx.org/contents/jQSmhtXo@16.36:R3te9Vua@4/52-Drag-Forces

![grey_capt](https://user-images.githubusercontent.com/40228854/43703803-34d0d674-995e-11e8-9faa-2db97c75328c.JPG)

This is how it looks in PDF:

![grey_capt_pdf](https://user-images.githubusercontent.com/40228854/43703822-41a9acae-995e-11e8-87c2-c7b95bd84d5e.JPG)

**Comment by PiotrKozlowski** [comment]
This table is inside `.os-table` and there is caption inside `.os-caption-container` but probably greyed caption should be visable?

**Comment by openstaxalina** [comment]
@wilkus27 is table 5.2 the **only** table you found like that in AP College Physics?  If so, I think Table 5.2 is a content issue.

@helenemccarron, note that Stefan's screenshot is from an Appendix.  Here is the same content in the PDF:
<img width="280" alt="screen shot 2018-08-06 at 8 32 21 pm" src="https://user-images.githubusercontent.com/7863733/43749327-46804a48-99b9-11e8-9246-d0cfc02b24af.png">

**I think we should treat this issue for tables specifically in Appendices.**  If Natalia's issue occurs only once, it is likely content, and I can create a separate issue in the content-managers milestone.  Here's the markup from CNX (non-baked) for Table 5.2:
<img width="563" alt="screen shot 2018-08-06 at 8 39 33 pm" src="https://user-images.githubusercontent.com/7863733/43749365-6c4b4d86-99b9-11e8-8169-c5d4c22acfa6.png">

**Comment by wilkus27** [comment]
@openstaxalina Up to the 13th module I found just this table.

**Comment by helenemccarron** [comment]
I think this is recipe related. If I remember correctly, AP physics isn't set to have both captions, only titles, thus the captions aren't handled. Are we supporting captions and titles in AP Physics, @openstaxalina

**Comment by openstaxalina** [comment]
I did some cleanup on this issue.  The issue for Table 5.2 in AP Physics has been moved to https://github.com/Connexions/content-managers/issues/216; it is a content issue.

This issue is now specifically about Table C7 and other appendices tables in College Physics.

**Comment by helenemccarron** [comment]
@openstaxalina I think https://katalyst01.cnx.org/contents/Ax2o07Ul@11.32:cjQyhWg-@7/Useful-Information is missing the `appendix` class. Notice how the appendix is not numbered properly in the navigation.

![2018-08-08_12-29-42](https://user-images.githubusercontent.com/8514591/43853845-e696bc94-9b06-11e8-9b2d-f35793f653eb.png)

**Comment by openstaxalina** [comment]
Thank you for noticing that!  Confirmed - table numbers are showing in the next appendix:
<img width="1295" alt="screen shot 2018-08-08 at 12 35 09 pm" src="https://user-images.githubusercontent.com/7863733/43854105-8d00a7ca-9b07-11e8-9f54-6fe193752c4b.png">

I will transfer this issue to content-managers.  cc @helenemccarron

**Comment by openstaxalina** [comment]
Issue moved to [Connexions/content-managers #218](https://github.com/Connexions/content-managers/issues/218) via [**ZenHub**](https://www.zenhub.com/)
